website/content/docs/intro/index.mdx

```diff
@@ -16,9 +16,19 @@ with Consul. We cover what Consul is, what problems it can solve, how it compare
 to existing software, and how you can get started using it. If you are familiar
 with the basics of Consul, the [documentation](/docs) provides a more
 detailed reference of available features. If you're ready to get hands-on
-experience, deploy Consul locally with our 
+experience, deploy Consul locally with our
 [HashiCorp Learn tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-install).
 
+## Why Consul?
+
+Consul solves the challenges that organizations of all sizes encounter with microservices architectures. This ranges from operating in various distributed environments and geographical locations, to meeting the need of securing all application traffic.
+The world is rapidly changing and evolving, so is the computing networking layer.
+
+Today's network must quickly adapt and ensure communication is encrypted at all times. Consul enables organizations to embrace a [zero trust](https://www.hashicorp.com/solutions/zero-trust-security) model while scaling up.
+Consul can achieve all this while reducing the burden on both operators and developers through automation of crucial networking tasks
+
```

Comment:
```suggestion
Consul solves the computing network layer challenges that organizations of all sizes face when implementing microservice architectures. These challenges range from operating in various distributed environments and geographical locations to securing all application traffic.

Modern networks must be highly adaptable while ensuring that communication is always encrypted. Consul enables organizations to embrace a [zero trust](https://www.hashicorp.com/solutions/zero-trust-security) model that scales to meet fluctuating workloads.
Consul achieves these goals while reducing the burden on operators and developers by automating crucial networking tasks.

```
Ignore this suggestion if it doesn't seem like an improvement --  experimenting with the language to see if we can improve the flow.